galasa-managers-parent/galasa-managers-zos-parent/dev.galasa.zos3270.common/src/test/java/dev/galasa/zos3270/common/screens/images/TerminalImageTransformTest.java

```diff
@@ -0,0 +1,721 @@
+                " As a result, this test cannot check the expected vs actual images rendered."
+            );
+        } else {
+            assertFileContentsSame( contentsRendered , testFileToCompareAgainst , imageFolderName );
+        }
+    }
+
+    private String getImageFolderName() { 
+        String folderName = null;
+
+        String osName = System.getProperty("os.name");
+
+        // We have mac images captured and stored waiting to be compared against.
+        if (osName == "Mac OS X") {
+            folderName = "mac";
+        }
+
+        // Other operating systems have different fonts, so the rendered images look different.
+
+        return folderName ;
+    }
+
+    private void assertFileContentsSame(
+        byte[] contentsRendered, 
+        String testFileToCompareAgainst,
+        String imageFolderName
+    ) throws IOException {
+
+        boolean isCheckedAgainstTestFile = false ;
+        try ( InputStream testImageToCompare = this.getClass().getClassLoader().getResourceAsStream(imageFolderName+"/"+testFileToCompareAgainst)
+        ) {
+            if (testImageToCompare == null) {
+                String tempFilePath = writeTempFile(testFileToCompareAgainst,contentsRendered);
+                System.out.println("Testcase logic info: Tried to open file "+testFileToCompareAgainst+
+                    " using this.getClass().getClassLoader().getResourceAsStream(...) but it was missing. "+
+                    "... was hoping to compare it to image\n"+tempFilePath);
+            } else {
+                System.out.println("Testcase logic info: Opened file "+testFileToCompareAgainst+
+                    " using this.getClass().getClassLoader().getResourceAsStream(...)");
+                isCheckedAgainstTestFile = true ;
+                assertInputStreamContentsSame(testImageToCompare, contentsRendered, testFileToCompareAgainst);
+            }
+        }
+
+        // try ( InputStream testImageToCompare = this.getClass().getResourceAsStream("/"+imageFolderName+"/"+testFileToCompareAgainst)
+        // ) {
+        //     if (testImageToCompare == null) {
+        //         String tempFilePath = writeTempFile(testFileToCompareAgainst,contentsRendered);
+        //         System.out.println("Testcase logic info: Tried to open file "+testFileToCompareAgainst+
+        //             " using this.getClass().getResourceAsStream('/'+...) but it was missing. "+
+        //             "... was hoping to compare it to image\n"+tempFilePath);
+        //     } else {
+        //         System.out.println("Testcase logic info: Opened file "+testFileToCompareAgainst+
+        //         " using this.getClass().getResourceAsStream('/'+...)");
+        //         isCheckedAgainstTestFile = true ;
+        //         assertInputStreamContentsSame(testImageToCompare, contentsRendered, testFileToCompareAgainst);
+        //     }
+        // }
```

Comment:
If we don't need this, maybe it can be removed?

Reply:
Removed.